packages/react-vapor/src/components/select/examples/SingleSelectExamples.tsx

```diff
@@ -35,6 +35,8 @@ export interface ISingleSelectExamplesState {
 }
 
 export class SingleSelectExamples extends React.PureComponent<{}, ISingleSelectExamplesState> {
+    static description = 'Single Selects allow to choose one single element among a list of two or more options.';

```

Comment:
> a list of two or more options

What about custom values? Isn't it legit to use a single select with custom values when there are no options initially but you can add new options gradually in the life span of the application?